common/src/main/java/org/dash/wallet/common/Configuration.java

```diff
@@ -437,10 +437,17 @@ public void setShowJoinDashPay(boolean showJoinDashPay) {
     }
 
     public long getLastSeenNotificationTime() {
-        return prefs.getLong(PREFS_LAST_SEEN_NOTIFICATION_TIME, 0);
+        long savedTime = prefs.getLong(PREFS_LAST_SEEN_NOTIFICATION_TIME, 0);
+        long distantFutureTime = 2524608000L; //2050-01-01
+        if (savedTime > distantFutureTime) {
+            // date incorrectly saved in microseconds, fix it
+            savedTime = savedTime / 1000;
+            setPrefsLastSeenNotificationTime(savedTime);
+        }
```

Comment:
This code can be removed as soon as we make sure all existing users (i.e. alpha-testers) updated to this version.

Reply:
We won't need this because after our recent updates in our code to v0.15 Platform, users in the alpha program can't update without starting from scratch.  The next version that they get will be on fresh new network.